testsuite/pytests/sli2py_regressions/test_ticket_156.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+#
+# test_ticket_156.py
+#
+# This file is part of NEST.
+#
+# Copyright (C) 2004 The NEST Initiative
+#
+# NEST is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# NEST is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+import nest
+import numpy as np
+import pytest
+
+
+def test_ticket_156():
+    """
+    Verify that poisson_generator_ps generates identical spike trains for given
+    start and stop times across different simulation resolutions and different targets.
```

Comment:
Perhaps clarify here how this relies on resetting the random state with ResetKernel